### PR TITLE
Layout

### PR DESCRIPTION
**Pull request by lubko43**

### CODE REVIEW

File: events/static/css/styles.less [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+html,body{
+  height:100%;
+  .container{
+    min-height: 100%;
+    position: relative;
+  }
+ footer{
+    position: relative;
+ }
+}
```
Comment (lubko43) [inline events/static/css/styles.less]: add end line

File: events/templates/base.html [hunk 2]
```diff
@@ -3,21 +3,32 @@
...
-    <h1>Hello world!</h1>
+    <div class="container">
+        <h1>Hello world!</h1>
+    </div>
 {% endblock content %}
+
+{% include 'footer.html' %}
+
 </body>
 </html>
```
Comment (lubko43) [inline events/templates/base.html]: add end line

**Comment by Scandie** [comment]
Server wouldn't start without installed lessc on developer's machine . Add proper requirements for this.

File: events/templates/base.html [hunk 3]
```diff
@@ -3,21 +3,32 @@
...
 
     {% block head %}{% endblock head %}
 
-    <link type="text/css" rel="stylesheet/less" href="{% static 'css/styles.less' %}" />
+    <link rel="stylesheet" href="{% static 'css/styles.less'|compile %}" />
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"
+          integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
 
     <script type="text/javascript" src="{% static 'js/react/react.js' %}"></script>
 	<script type="text/javascript" src="{% static 'js/react/react-dom.js' %}"></script>
```
Comment (Scandie) [inline events/templates/base.html:16]: @lubko43 fix the indentation

**Comment by VKotsovskyi** [comment]
+1 Looks good for me.

**Review by VKotsovskyi — APPROVED** [review]

**Review by Scandie — APPROVED** [review]

**Pull request merged**